addons/medical/functions/fnc_handleFallDamage.sqf

```diff
@@ -0,0 +1,14 @@
+// by commy2
+#include "script_component.hpp"
+
+params ["_unit", "_newDamage"];
+
+private ["_totalDamageL", "_totalDamageR"];
+
+_totalDamageL = (_unit getHitPointDamage "HitLeftLeg") + _newDamage;
+_totalDamageR = (_unit getHitPointDamage "HitRightLeg") + _newDamage;
+
+_unit setHitPointDamage ["HitLeftLeg", _totalDamageL];
+_unit setHitPointDamage ["HitRightLeg", _totalDamageR];
```

Comment:
Wrong. Those are custom hitpoints we add ourselves. Same as `HitLeftArm` and `HitRightArm`.